README.md

```diff
@@ -6,11 +6,78 @@ implemented in pure Go, i.e. does not wrap libnftnl.
 
 This is not an official Google product.
 
-## Breaking changes
+## Alpha status
```

Comment:
Can you keep the “Breaking changes” in the heading please? I’d like to have it prominently pointed out.